metals/src/main/scala/scala/meta/metals/Main.scala

```diff
@@ -15,20 +15,69 @@ import scala.meta.internal.metals.clients.language.MetalsLanguageClient
 import scala.meta.internal.metals.logging.MetalsLogger
 
 import org.eclipse.lsp4j.jsonrpc.Launcher
+import coursierapi.Complete
+import scala.jdk.CollectionConverters._
+import scala.util.Try
 
 object Main {
   def main(args: Array[String]): Unit = {
     if (args.exists(Set("-v", "--version", "-version"))) {
+      val api: Complete = coursierapi.Complete
+        .create()
+        .withInput("org.scalameta:mtags")
+
+      val supported =
+        BuildInfo.supportedScalaVersions.toSet
+
+      import scala.collection.parallel._
+
+      val additional =
```

Comment:
Maybe we can wrap it in `Future` and do something like
```scala
val additional = 
  try Await.result(additionalFuture, 10.seconds)
  catch {
    case _: Throwable => Nil
  }
```
So when we don't get response from coursier we still show other supported versions?